components/Stepper/helper.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+const handlerValuePrepare = total => {
+  if (total > 6 && total > 0) return 6;
+  if (total < 1) return 1;
```

Comment:
this value `6` couldn't be dynamic? Because I would like to create more steps.

Reply:
This maximum number was asked by the ux team. They tested it and arrived at this maximum number, we came to the conclusion that it doesn't make sense so many steps for a form, for example.
For now, we keep 6 for a maximum steps. In case of you need more steps, talk with them to make new test cases with more steps.